strangerstudios/paid-memberships-pro
# Fix PHP notice issues in name parser library

### All Submissions:






### Changes proposed in this Pull Request:

This PR solves issues reported in https://wordpress.org/support/topic/can-you-fix-this-error/

### How to test the changes in this Pull Request:

1. Check out with a name that is one word
2. See no PHP notices during any of that process

### Other information:




### Changelog entry

> Resolve issues with PHP notices when using one-word names
